src/Groups/group_characters.jl

```diff
@@ -282,6 +282,18 @@ function character_table(series::Symbol, parameter::Union{Int, Vector{Int}})
     return tbl
 end
 
+
+# For character tables with stored group, we take the hash value of the group.
+# For character tables without stored group, we take the table identifier.
+function Base.hash(tbl::GAPGroupCharacterTable, h::UInt)
```

Comment:
shouldn't you proved a default for the UInt as to also allow `hash(tbl)`?

Reply:
No, there should not be a default. There is a fallback.